azure/batch/scripts/predict_on_catalog.py

```diff
@@ -104,39 +106,107 @@ def setup(self, stage: Optional[str] = None):
+        }
       }
+      # will add the actual data under 'data' key, below
     }
+
+    # check that the predictions mean what we think they mean
+    assert label_cols[0] == 'smooth-or-featured-cd_smooth'
+    assert label_cols[1] =='smooth-or-featured-cd_featured-or-disk'
+    assert label_cols[2] == 'smooth-or-featured-cd_problem'
+    # okay, now it's safe to hardcode the values below
+    
     # only derive each galaxies smooth or features question right now for simplicity of metric
     # i.e. we're trying to figure out if this galaxy is interesting or not for human volunteers
-    # if it's smooth it's not interesting so we can use this metric to decide to show it to volunteers
+    # if it's not featured it's not interesting so we can use this metric to decide to show it to volunteers
     smooth_or_featured_start_and_end_indices = [0, 2]
-    # the smooth answer label index
-    smooth_or_featured_smooth_index = 0
-    # lower bound of volunteers answering for a feature, i.e. 20% of volunteers say yes to the feature
-    odds_bound = 0.2
-    #
-    # variances = predictions_to_variance_of_answer(predictions, smooth_or_featured_start_and_end_indices, smooth_or_featured_smooth_index)
-    # expectations = predictions_to_expectation_of_answer(predictions, smooth_or_featured_start_and_end_indices, smooth_or_featured_smooth_index)
-    probability_galaxy_is_not_smooth = odds_answer_below_bounds(predictions,
+
+    # the featured answer label index
+    smooth_or_featured_featured_index = 1
+
+    # upper bound of volunteers answering for a feature, i.e. no more than e.g. 20% of volunteers select the answer (here, featured)
+    featured_upper_bound = 0.2
+
+    # currently, probability volunteers would give featured vote fraction below 20%
+    probability_volunteers_say_featured_below_bound = odds_answer_below_bounds(predictions,
         smooth_or_featured_start_and_end_indices,
-        smooth_or_featured_smooth_index,
-        odds_bound
+        smooth_or_featured_featured_index,
+        featured_upper_bound
     )
-    # output the probability data as subject_id: percentage probability galaxy is not smooth (i.e. has features / interesting!)
-    prediction_data = {id_str[n]: round((probability_galaxy_is_not_smooth[n][0] * 100.0), 4) for n in range(len(predictions))}
+
+    # just for convention and for the existing kade code, it seems more natural to have high probability galaxies be put in the active set
+    # https://github.com/zooniverse/kade/blob/9413312ba9629bd256426eb400e06c47e8d0968f/app/services/prediction_results/process.rb#L33
+    # so let's record the odds that volunteers say featured will be *above* the bound (i.e. at least 20%)
+    probability_volunteers_say_featured_above_bound = 1 - probability_volunteers_say_featured_below_bound
+
+    # output the probability data as subject_id: probability volunteers say featured above bound (rounded to 4dp)
+    # note - no longer a percentage probability
```

Comment:
How does this change impact processing the results in KaDE, i.e. will it the threshold comparison still work the same way? 
https://github.com/zooniverse/kade/blob/9413312ba9629bd256426eb400e06c47e8d0968f/app/services/prediction_results/process.rb#L35-L36